prisma/utils/apply-views.ts

```diff
@@ -0,0 +1,65 @@
+import { execSync } from "child_process";
+import dotenv from "dotenv";
+import fs from "fs";
+dotenv.config();
+
+const viewFiles = fs.readdirSync("./prisma/views");
+console.log("Creating views...");
+console.log("DATABASE_URL", process.env.DATABASE_URL);
+
+// Get db URL
+const dbUrl = process.env.DATABASE_URL || "";
+const psqlUrl = (() => {
+  try {
+    const u = new URL(dbUrl);
```

Comment:
```suggestion
    const url = new URL(dbUrl);
```
Je préfère éviter les noms de variables à une lettre pour des raisons de lisibilité